kitsune/wiki/tasks.py

```diff
@@ -39,15 +40,16 @@
 
 
 @shared_task
-def send_reviewed_notification(revision_id: int, document_id: int, message: str):
+def send_reviewed_notification(revision_id: int, message: str):
     """Send notification of review to the revision creator."""
 
     try:
-        revision = Revision.objects.get(id=revision_id)
-        document = Document.objects.get(id=document_id)
-    except (Revision.DoesNotExist, Document.DoesNotExist) as err:
+        revision = Revision.objects.select_related("document").get(id=revision_id)
+    except Revision.DoesNotExist as err:
         capture_exception(err)
         return
+    else:
+        document = revision.document
```

Comment:
Is there a case where this is not true and we need to pass a the document id? (revision.document.id != document_id from the function signature)